apiserver/migrationtarget/migrationtarget.go

```diff
@@ -24,13 +24,20 @@ import (
 // API implements the API required for the model migration
 // master worker when communicating with the target controller.
 type API struct {
+	caCerter
 	state      *state.State
 	authorizer facade.Authorizer
 	resources  facade.Resources
 	pool       *state.StatePool
 	getEnviron stateenvirons.NewEnvironFunc
 }
 
+// caCerter implements the subset of common.APIAddresser
```

Comment:
The name would kind of suggest that it will only contain the one function, if that's not the case then maybe the interface should be named differently.